### DISCUSSION TITLE
Editing cards during the review

### DISCUSSION THREAD

**Discussion by myrnaw** [st3v3nmw/obsidian-spaced-repetition#267] [General]
Hi,
Thank you for making this awesome plugin!
I know we can jump to the file by clicking the link that shows in the card, and then edit it. But it's more smooth editing during the review.
Is it possible to support this feature?

**Comment by st3v3nmw** [comment]
It's possible, you could open an issue for this for better tracking. Might take a while though.

**Reply by myrnaw** [reply]
that's cool! thank you!
